etc/config/cobol.defaults.properties

```diff
@@ -0,0 +1,15 @@
+# Default settings for COBOL
+compilers=&gnucobol
+defaultCompiler=gnucoboldefault
+compilerType=gnucobol
```

Comment:
```suggestion
compilerType=cobol
```
this is needed to get the server to start up

Reply:
Too bad we have a lang name used to denote a compiler... Only brings confusion later on, when we have several compilers for the same lang :'(

Reply:
we can rename the compilerType instead